flint.ui/src/views/gcbm/GCBMDashboard.vue

```diff
@@ -1,27 +1,26 @@
 <template>
   <div>
     <div class="flex flex-wrap">
-      <div class="relative md:pt-32 pb-32 pt-12 w-full h-auto">
-        <LandingPageNavbar />
-        <div class="px-4 md:px-10 mx-auto w-full">
-          <div class="bg-white p-6 rounded-lg shadow-lg">
-            <h2 class="mt-3 text-2xl font-bold mb-2 text-gray-800">GCBM simulation workflow</h2>
-            <p class="text-gray-700">Follow the steps below to simulate GCBM runs.</p>
-          </div>
+        <div class="relative w-full h-auto">
```

Comment:
The spacing of tags in this file is not correct, can you please fix this?
Prettier extension can automatically fix the formatting for you, also install Prettier ESLint, ESLint if you haven't already in your code editor. 
You can also set "format on save" option to true if you're using vscode, so that the code is auto-formatted as you save the file.

Reply:
@YashKandalkar Thank you for pointing. I will keep these things in mind from now onwards. I will send the updated PR with the prettier soon.